app/src/main/java/org/xbmc/kore/ui/MovieDetailsFragment.java

```diff
@@ -508,7 +546,7 @@ private void displayMovieDetails(Cursor cursor) {
                 movieTitle, cursor.getString(MovieDetailsQuery.FILE));
 
         // Check if downloaded file exists
-        downloadButton.setVisibility(View.VISIBLE);
+        downloadButton.setEnabled(true);
```

Comment:
Can't remember why the downloadButton was `gone` and only turned visible here... Can't think of a good justification for that, maybe it was just an oversight.
If that was the case, there's no need to do the disable/enable of it. Just leaving it be always on should work fine.